### PR TITLE
refactor : plottly.js integration + removed select and icons + fixed logo height

### PR DESCRIPTION
**Pull request by ACU017**
This PR is a recap of : 

#26 
- Integrating Plottly.js into the DummyFocus2.tsx 
- Removing the background from the plot 

#33  Refactor 
- Removed the select 
- Removed the icons and their links
- Commented the search bar just in case we decide to use it in the future. 

Extra: Checked the onrender logo css to see if it compares to ours and made it bigger

### CODE REVIEW

**Comment by irenezatti** [comment]
This looks good Andrea! And thanks for checking for the size of the logo too, that was needed indeed

**Comment by irenezatti** [comment]
You can go ahead and merge it when you're ready

**Comment by sssoja** [comment]
Great PR description @ACU017 - i've left you a comment in Slack regarding the logo. 

Please use the newer versions of the EuFMD logo, you can choose the black or the white one, whichever you think looks best.

File: client/src/components/Navbar.tsx [hunk 1]
```diff
@@ -21,35 +16,20 @@ const Navbar: React.FC = () => {
             <img
               src={logo}
               alt="FAO Logo"
-              className="h-8"
+              className="h-40"
+              // these are the setting from the on render logo
+              //class="w-16 md:w-28 h-auto my-2 md:my-4"
               onClick={takeToHome}
             />
```
Comment (sssoja) [inline client/src/components/Navbar.tsx:23]: You don't need the `onClick` handler here as React has an in-built `Link` component. 

```
  {/* logo */}
          <div className="flex items-center space-x-4">
            <Link to="/"> {/* Wrap the logo in a Link component */}
              <img
                src={logo}
                alt="FAO Logo"
                className="h-40"
              />
            </Link>
          </div>
```

**Review by sssoja — CHANGES_REQUESTED** [review]

File: client/src/components/Navbar.tsx (hunk 1 above)
Comment (ACU017) [reply client/src/components/Navbar.tsx]: Done Changed it and checked that it works

**Pull request merged**